examples/tv-app/tv-common/clusters/channel/ChannelManager.cpp

```diff
@@ -60,6 +60,17 @@ ChannelManager::ChannelManager()
     worldChannel.minorNumber       = static_cast<uint16_t>(3);
     mChannels.push_back(worldChannel);
 
+    ChannelInfoType ottChannel;
+    ottChannel.affiliateCallSign = MakeOptional(chip::CharSpan::fromCharString("OTT"));
+    ottChannel.callSign          = MakeOptional(chip::CharSpan::fromCharString("OTT-TV"));
+    ottChannel.name              = MakeOptional(chip::CharSpan::fromCharString("OTT Channel"));
+    ottChannel.majorNumber       = static_cast<uint8_t>(0);
+    ottChannel.minorNumber       = static_cast<uint16_t>(0);
+    ottChannel.identifier        = MakeOptional(chip::CharSpan::fromCharString("ott-1"));
+    // ottChannel.type              = MakeOptional(chip::app::Clusters::Channel::ChannelTypeEnum::kUnknownEnumValue); // TODO: OTT
```

Comment:
TODO: More description?

Reply:
issue https://github.com/project-chip/connectedhomeip/issues/31320 is tracking the OTT gap referred to here. However, I need to remove this channel from the example because there are CI test cases which assume there are only 3 channels.